sdks/python/apache_beam/io/gcp/spanner_wrapper_test.py

```diff
@@ -0,0 +1,64 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the 'License'); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import mock
+import unittest
+from unittest.mock import MagicMock
+
+from apache_beam.io.gcp.spanner_wrapper import SpannerWrapper
+
+
+class TestSpannerWrapper(unittest.TestCase):
+  @mock.patch('apache_beam.io.gcp.spanner_wrapper.spanner.Client')
+  def setUp(self, mock_client):
+    self.mock_client = mock_client
+    self.wrapper = SpannerWrapper(project_id='test-project')
+    self.mock_instance = MagicMock()
+    self.mock_database = MagicMock()
+    self.mock_client.instance.return_value = self.mock_instance
+    self.mock_instance.database.return_value = self.mock_database
+    self.wrapper._spanner_client = self.mock_client
+    self.wrapper._spanner_instance = self.mock_instance
+    self.wrapper._test_database = self.mock_database
```

Comment:
Try just creating an __init__ method instead. (`@mock.patch` is intended to annotate a class that implements the one being mock - which is not what is happening here)
```suggestion
  def __init__(self):
    self.wrapper = SpannerWrapper(project_id='test-project')
    self.mock_instance = MagicMock()
    self.mock_database = MagicMock()
    self.mock_client.instance.return_value = self.mock_instance
    self.mock_instance.database.return_value = self.mock_database
    self.wrapper._spanner_client = mock.Mock()
    self.wrapper._spanner_instance = self.mock_instance
    self.wrapper._test_database = self.mock_database
```